File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.py
class Solution:
    def getXORSum(self, arr1: List[int], arr2: List[int]) -> int:
        xorArr2 = 0;
        for i in arr2: xorArr2 ^= i
        xorArr1 = 0
        for j in arr1: xorArr1 ^= j
        return xorArr2 & xorArr1

File: 1835-find-xor-sum-of-all-pairs-bitwise-and/1835-find-xor-sum-of-all-pairs-bitwise-and.java
class Solution {
    public int getXORSum(int[] arr1, int[] arr2) {
        int xorArr2 = 0;
        for(int i : arr2) xorArr2 ^= i;
        int xorArr1 = 0;
        for(int j : arr1) xorArr1 ^= j;
        return xorArr2 & xorArr1;
    }
}